nixos/tests/fwupd.nix

```diff
@@ -16,6 +16,6 @@ import ./make-test.nix ({ pkgs, ... }: {
   };
 
   testScript = ''
-    $machine->succeed("gnome-desktop-testing-runner");
+    machine.succeed("gnome-desktop-testing-runner")
```

Comment:
We should probably standardize on passing the installed test dir to the command instead of setting `XDG_DATA_DIRS`.

Reply:
Sure, I can incorporate that into the testing function.